src/roller/table.ts

```diff
@@ -186,14 +185,14 @@ export class TableRoller extends GenericFileRoller<string> {
     }
 }
 const MATCH = /^\|?([\s\S]+?)\|?$/;
-const SPLIT = /\|/;
+const SPLIT = /(?<!\\)\|/g;
```

Comment:
Unfortunately look behinds are not supported by WebKit, which means this won’t work on iOS mobile Obsidian.

Reply:
Oh, too bad....

So back to doing it manually ;-)